harvester_e2e_tests/apis/test_images.py

```diff
@@ -19,6 +19,7 @@
 from datetime import datetime, timedelta
 
 import pytest
+import os
```

Comment:
standard library should be placed in the first group.[^pep8]

[^pep8]: https://peps.python.org/pep-0008/#imports

Reply:
Update related import os to the first group in integration/test_images.py